src/View/Home/index.html.twig

```diff
@@ -1,5 +1,72 @@
-{% extends 'layout.html.twig' %}
+{% extends 'sublayout_visitor.html.twig' %}
 
 {% block content %}
-    <h1>Hello Wilder !</h1>
+
+    <section id="HarticleClub">
+        <div class="container-fluid">
+            <div class="text-center"><h2>Actualité du club</h2></div>
+            <div class="row">
+                <div class="col-md-6">
+                    <a href="#">
+                        <div class="big_club_article">
+                            <div class="info_article_big">
+                                <ul>
+                                    <li>
+                                        <i class="fas fa-calendar-alt"><span>{{ article[0].date }}</span></i>
```

Comment:
toujours non
1. je n'ai tjs pas de récup des articles en BDD, ce qui devrait se faire dans cette PR. Ton controller devrait ensuite envoyer une variable articles à ton twig
2. dans le twig, il faut une boucle sur {% for article in articles %} et pas reprendre article0, article1 etc